src/main/java/redis/clients/jedis/ConnectionFactory.java

```diff
@@ -76,7 +76,7 @@ public boolean validateObject(PooledObject<Connection> pooledConnection) {
       // check HostAndPort ??
       return jedis.isConnected() && jedis.ping();
     } catch (final Exception e) {
-      logger.error("Error while validating pooled Connection object.", e);
+      logger.warn("Error while validating pooled Connection object.", e);
```

Comment:
I think general business will configure the level of info. There seems to be no difference between error or warn (both will be printed). Maybe we should consider debug?